include/sparrow/utils/nullable.hpp

```diff
@@ -146,65 +156,60 @@ namespace sparrow
 
         template <class T, class Arg>
-        concept initializable_from_refs =
-            initializable_from_one<T, Arg&, const Arg&, Arg&&, const Arg&&>;
+        concept initializable_from_refs = initializable_from_one<T, Arg&, const Arg&, Arg&&, const Arg&&>;
 
         // We prefer std::is_assignable_v to std::assignable_from<To, From> because
         // std::assignable_from requires the existence of an implicit conversion
         // from From to To
         template <class To, class... Args>
-        concept assignable_from_one =
-            (std::is_assignable_v<std::add_lvalue_reference<To>, Args> && ...);
+        concept assignable_from_one = (std::is_assignable_v<std::add_lvalue_reference<To>, Args> && ...);
 
         template <class T, class Arg>
-        concept assignable_from_refs =
-            assignable_from_one<T, Arg&, const Arg&, Arg&&, const Arg&&>;
+        concept assignable_from_refs = assignable_from_one<T, Arg&, const Arg&, Arg&&, const Arg&&>;
 
         template <class T, class U>
         using conditional_ref_t = std::conditional_t<std::is_reference_v<T>, const std::decay_t<U>&, std::decay_t<U>&&>;
 
         template <class T, class Targs, class U, class UArgs>
-        concept both_constructible_from_cond_ref =
-            std::constructible_from<T, conditional_ref_t<T, Targs>> and
-            std::constructible_from<U, conditional_ref_t<U, UArgs>>;
+        concept both_constructible_from_cond_ref = std::constructible_from<T, conditional_ref_t<T, Targs>>
+                                                   and std::constructible_from<U, conditional_ref_t<U, UArgs>>;
 
         template <class To1, class From1, class To2, class From2>
-        concept both_convertible_from_cond_ref =
-            std::convertible_to<conditional_ref_t<To1, From1>, To1> and
-            std::convertible_to<conditional_ref_t<To2, From2>, To2>;
+        concept both_convertible_from_cond_ref = std::convertible_to<conditional_ref_t<To1, From1>, To1>
+                                                 and std::convertible_to<conditional_ref_t<To2, From2>, To2>;
 
         template <class To1, class From1, class To2, class From2>
-        concept both_assignable_from_cref =
-            std::is_assignable_v<std::add_lvalue_reference_t<To1>, mpl::add_const_lvalue_reference_t<From1>> and
-            std::is_assignable_v<std::add_lvalue_reference_t<To2>, mpl::add_const_lvalue_reference_t<From2>>;
+        concept both_assignable_from_cref = std::is_assignable_v<
+                                                std::add_lvalue_reference_t<To1>,
+                                                mpl::add_const_lvalue_reference_t<From1>>
+                                            and std::is_assignable_v<
+                                                std::add_lvalue_reference_t<To2>,
+                                                mpl::add_const_lvalue_reference_t<From2>>;
```

Comment:
I find this version harder to read than the original one (but that should be fixed in the formatting config, not in that PR).